docs/access-the-swarm/direct-upload.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Direct upload
+id: direct-upload
+---
+
+:::caution
+We recomment turning the encryption *ON* when using this feature. Otherwise the uniformity assumption will be broken.
+If the encryption is *OFF* the stamp value will decrease faster.
```

Comment:
this is true only if you try to reupload the same content again using the same postage stamp